src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -275,5 +233,8 @@ export default compose(
         reimbursementAccount: {
             key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
         },
+        validationStepForm: {
```

Comment:
Q: Where are we using this `prop` in `ValidationStep` component? Shouldn't we add it to `propTypes` and `defaultProps`?

Reply:
Hmm we're not using it in the file at all, so I'll remove it 👍🏻